### PR TITLE
Fixed grammar in Applications and AT

### PR DESCRIPTION
**Pull request by WilliamStarling**
Fixed minor grammar and spelling issues in the Applications and Algebraic Properties of Linear Maps (AT) sections, with the help of the Grammarly writing tool.

### CODE REVIEW

File: source/linear-algebra/source/03-AT/04.ptx
Comment (StevenClontz) [inline source/linear-algebra/source/03-AT/04.ptx:1041]:
```suggestion
                    <li>Prove that <m>f</m> is one-to-one if and only if <m>f(\vec{0_V}) = \vec{0_W}</m>, and that <m>\vec{0_V}</m> is the unique element of <m>V</m> which is mapped to <m>\vec{0_W}</m>. Remember that this needs to be done in both directions. First, prove the if and only if statement, and then show the uniqueness.</li>
```

**Review by StevenClontz — COMMENTED** [review]
Thanks @WilliamStarling! I'm leaving this to another contributor to approve. I did make one suggested change.

**Review by jkostiuk — APPROVED** [review]
looks good to me, thanks for the fixes!

**Comment by StevenClontz** [comment]
@jkostiuk - there was a small tweak I recommended above - we can't merge it in with this PR but if you approve then I can make a quick PR to implement it.

**Comment by jkostiuk** [comment]
sounds good, i just commented that I approve. what was the "correct" workflow to have incorporated your changes and then approved?

**Comment by StevenClontz** [comment]
You also pressed the "Squash and merge" button, which is normally the right thing to do. But since I proposed a change at https://github.com/TeamBasedInquiryLearning/library/pull/441#discussion_r1874561935 we should have first decided whether to implement the change (by clicking "commit suggestion") before the merge. Note the "commit suggestion" button doesn't appear now that it's too late to make the change here, but I'll make a new PR with the tweak for you to approve and merge.

**Pull request merged**
